wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,208 @@
--   **V**: [Spun-Out](/wiki/Game_Modifiers)
--   **B**: Auto
-
-Play Mode
----------
-
--   **+/-**: Adjust local offset (hold Alt to change the precision)
--   **Escape**: Pause (Quit song if Auto mod is used)
--   **Spacebar**: Skip opening cut-scene.
--   **Tab**: Toggle scoreboard
--   **Shift+Tab**: Toggle in-game interface
--   **F3/F4**: Adjust speed for osu!mania.
--   **Ctrl+R or \`**: Quick retry the current map (autoskips the cut-scene)
-
-Edit Mode
----------
-
-### Keyboard Shortcuts
-
-#### General
-
-![How it would look like when you press H (Help) key](Edit_Help.jpg "How it would look like when you press H <Help> key")
-
--   **H**: Help (Contains all the shortcuts shown below)
-
-**Music**
-
--   **Z**: Stop
--   **X**: Play from start
--   **C or Spacebar**: Pause/Resume
--   **Left and Right arrow keys**: Skip through the song. Hold shift for 4x speed
-
-**Bookmarks commands**
-
--   **Ctrl+B**: Add bookmark at current location.
--   **Ctrl+Shift+B**: Remove bookmark at current location.
--   **Ctrl+Left Arrow**: Jump to previous bookmark.
--   **Ctrl+Right Arrow**: Jump to next bookmark.
-
-**Save and Reset**
-
--   **Ctrl+L**: Revert to saved
--   **Ctrl+S**: Quick save
--   **Ctrl+N**: Remove all notes (but keep timing/song setup)
-
-**Mode switch**
-
--   **F1**: Compose mode
--   **F2**: Design mode (only basics available at the moment)
--   **F3**: Timing mode
--   **F4**: Song setup
-
-#### Compose mode
-
--   **Left and Right arrow keys**: Move selected notes by 1 grid snap. Hold shift for 1 coordinate.
--   **Number keys 1-4**: Change selection/placement mode
--   **Q,W,E,R,L**: Change right-hand option modes
--   **Delete**: Remove selected notes
--   **Ctrl+Z,Y**: Undo,Redo
--   **Ctrl+A**: Select all notes
--   **Ctrl+X,C,V**: Cut, copy, paste
--   **Ctrl+D**: Clone the selection
--   **Alt while placing**: Snap distance based on time duration since last note
--   **Shift while placing**: No grid snapping
--   **Shift while modifying**: No grid or beat (time) snapping
--   **J/K**: Nudge circle backwards/forwards in time
--   **G**: Cycle through grid precision
--   **Ctrl+1/2/3/4**: Grid sizes (tiny,small,medium,large)
--   **Ctrl+Shift+R**: Rotate existing note by a specified angle
--   **Ctrl+ , (comma)**: Rotate Anticlockwise/Clockwise
--   **Ctrl+H,J**: Flip Horizontally/Vertically
-
-#### Timing mode
-
--   **T**: Tapping using the keyboard
--   **Shift while adjusting BPM/Offset**: 4x speed
--   **Ctrl+B**: Reset current timing point
--   **Ctrl+P**: Add timing point
+# Shortcut Key Reference
+
+There are many shortcuts in _osu!_ that can be utilized while in-game or while idle at any screen.
+They are listed out with their functions below.
+
+**Note:** some of these keys can be changed in the [Options](/wiki/Options/)!
+However, the list below presents the default keys.
+
+## Game-Wide
+
+These shortcuts work anywhere:
+
+- `Arrow Down`: Decrease volume.
+- `Arrow Up`: Increase volume.
```

Comment:
~~idk, it would either be make the page longer than it _could_ be or make it harder/more confusing than it _should_ be~~ hold on, reviewing and reconsidering on layout